cmd/clairctl/docker/login.go

```diff
@@ -0,0 +1,47 @@
+package docker
```

Comment:
First of all I was not thinking to get the docker library inside hyperclair for size matter, but definitely it would be better